tests/drivers/build_all/interrupt_controller/src/main.c

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright (c) 2012-2014 Wind River Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+int main(void)
+{
+	return 0;
```

Comment:
Could maybe add `BUILD_ASSERT()`s here to evaluate against DT

Reply:
added above